app/views/admin/detailed_guides/_form.html.erb

```diff
@@ -23,6 +23,11 @@
           label: "Related guides",
           heading_size: "m",
           options: taggable_detailed_guides_container(edition.related_detailed_guide_ids),
+          ga_data: {
```

Comment:
I don't really see why this is in the last commit and not the penultimate commit 🤔